apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/account/Account.java

```diff
@@ -467,7 +467,7 @@ public static boolean setOrVerify(long accountId, byte[] key) {
     }
 
     static void checkBalance(long accountId, long confirmed, long unconfirmed) {
-        if (accountId == Genesis.CREATOR_ID) {
+        if (accountId == GenesisImporter.CREATOR_ID) {
```

Comment:
Currently that doesn't make sense, because there are several classes which are not CDI components and it will be hard to 'inject' new GenesisImporter into them.